app/views/storage.html

```diff
@@ -55,7 +55,9 @@
                 </tbody>
                 <tbody ng-if="(pvcs | hashSize) > 0">
                   <tr ng-repeat="pvc in pvcs | orderObjectsByDate : true">
-                    <td data-title="Name"><a ng-href="{{pvc | navigateResourceURL}}">{{pvc.metadata.name}}</a></td>
+                    <td data-title="Name"><a ng-href="{{pvc | navigateResourceURL}}">{{pvc.metadata.name}}</a>
+                      <span ng-if="pvc | annotation : 'volume.beta.kubernetes.io/storage-class'" class="text-muted"> using storage class {{pvc | annotation : 'volume.beta.kubernetes.io/storage-class'}}</span>
```

Comment:
If we expect the `beta` to eventually drop from the annotation name or storage class to ever be added as another API field, it's probably worth adding a `storageClass` filter. Then we can change it in one place (and fall back to looking for this annotation if needed). Something like we do for the description annotation.

https://github.com/openshift/origin-web-console/blob/master/app/scripts/filters/resources.js#L94-L100

```js
  .filter('storageClass', function(annotationFilter) {
    return function(pvc) {
      return annotationFilter(pvc, 'volume.beta.kubernetes.io/storage-class');
    };
  })
```

Then here in the view

```html
 <span ng-if="pvc | storageClass" class="text-muted"> using storage class {{pvc | storageClass}}</span>
```

This also makes the markup more concise.